components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -982,7 +984,8 @@ enum TransactionStatus {
   Submitted = 3,
   Confirmed = 4,
   Error = 5,
-  Dropped = 6
+  Dropped = 6,
+  Signed = 7 // dapps will submit the transaction
```

Comment:
Does this need to be flagged to iOS or Android for handling in any switches?For Android is there anything in this PR that is needed?

Reply:
For Desktop, do we need anything in any of these which have other switch cases? 
- `components/brave_wallet_ui/components/shared/style.tsx`
- `components/brave_wallet_ui/components/extension/confirm-transaction-panel/confirm-solana-transaction-panel.tsx`
- `components/brave_wallet_ui/components/desktop/portfolio-transaction-item/index.tsx` (You have one covered in this file but there's another place)

I just did `git grep \\.Dropped` to get this list.

Reply:
Yes, iOS and Android need to handle the new tx status on UI. cc @SergeyZhukovsky @StephenHeaps.
`git grep TransactionStatus\\.` is more precise so you can see what is being used on Android.
I will check desktop UI for missing parts.

Reply:
I don't see other changes needed on UI except `components/brave_wallet_ui/components/shared/style.tsx` to update its opacity

Reply:
I will open two issues for iOS and Android as follow-up

Reply:
https://github.com/brave/brave-browser/issues/27224 and https://github.com/brave/brave-ios/issues/6606